.storybook/ussfTheme.js

```diff
@@ -0,0 +1,10 @@
+import { create } from '@storybook/theming'
+
+export default create({
+  base: 'dark',
+  brandTitle: 'USSF Design System',
+  brandUrl: 'https://github.com/USSF-ORBIT/ussf-portal-client',
+  brandImage: '/img/ussf-logo-short.svg',
+  brandTarget: '_self',
+  fontBase: '"Trebuchet MS", sans-serif',
```

Comment:
Question: Should we also use `Sharp Sans` like we do in the app or is that not necessary here for storybook themes?

Reply:
I hadn't originally because Storybook docs seemed to portray it as something possibly difficult (loading the font files into the root <head>), but I found a practical solution this morning that will appear soon.